vendor/github.com/grantseltzer/golumns/columns.go

```diff
@@ -62,5 +63,9 @@ func numberOfRows(numberOfRows, numberOfNames int) int {
 }
 
 func reverse(input []int) []int {
```

Comment:
Why not just use `sort.Reverse` instead of your own helper? https://golang.org/pkg/sort/#Reverse